docs/kyma/docs/04-04-gke-installation.md

```diff
@@ -7,13 +7,12 @@ This Installation guide shows developers how to quickly deploy Kyma on a [Google
 By default, Kyma is installed on a GKE cluster with a wildcard DNS provided by [xip.io](http://xip.io). Alternatively, you can provide your own domain for the cluster. 
 
 ## Prerequisites
-- [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project
-- [Kubernetes](https://kubernetes.io/) 1.12
-- Tiller 2.10.0 or higher
+- [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.0
 - [Docker](https://www.docker.com/)
-- [Docker Hub](https://hub.docker.com/) account
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
-- A domain for your GKE cluster (optional)
+- [wget](https://www.gnu.org/software/wget/)
+- A domain for your GKE cluster. You can get one for free using services like https://www.freenom.com (optional)
```

Comment:
Is this necessary? The installation defaults to xip.io as the domain provider so anyone can deploy on a cluster. I'd stick to the old formula of this prerequisite item.

Reply:
it actually do not defaults to xip, as there are also instructions about installing using own domain.
`A domain for your GKE cluster (optional)` was in the instruction already, I just added info about free service as it is very helpful, because I knew freenom from others I spent much less time on the installation

Reply:
Ok, I think this is a good opportunity to use our new **TIP** panel here.